README.md

```diff
@@ -83,7 +83,7 @@ Note that `agentai` automatically parses the Python Enum type (TemperatureUnit)
 3. **Create a Conversation object and add messages**
 
 ```python
-conversation = Conversation()
+conversation = Conversation(model=GPT_MODEL)
```

Comment:
I've not formed a full opinion on whether Conversation should take a GPT_MODEL param at all — I'm inclined to believe it should be either more general e.g. create a LLM class and build around that OR (more preferred) — read this from env in some way. 

In the mean time, merging this so that I can play with it and form stronger opinions.